### DISCUSSION TITLE
Welcome to SLICES Discussions!

### DISCUSSION THREAD

**Discussion by xiaohang007** [xiaohang007/SLICES#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gmrigna** [comment]
Dear Hang,
I have recently become aware of your work. First of all, let me congratulate you! It is really nice. It is the first very meaningful reconstruction rate!!
I have installed it on my mac (despite the warning because I would like to discover first how the graphs part works) but I have an error message which seems to be related to M3GNet:
`File /opt/homebrew/anaconda3/envs/slices_env/lib/python3.11/site-packages/keras/src/saving/saving_api.py:237, in load_weights(model, filepath, skip_mismatch, **kwargs)
    235             legacy_h5_format.load_weights_from_hdf5_group(f, model)
    236 else:
--> 237     raise ValueError(
    238         f"File format not supported: filepath={filepath}. "
    239         "Keras 3 only supports V3 `.keras` and `.weights.h5` "
    240         "files, or legacy V1/V2 `.h5` files."
    241     )

ValueError: File format not supported: filepath=/opt/homebrew/anaconda3/envs/slices_env/lib/python3.11/site-packages/m3gnet/models/MP-2021.2.8-EFS/m3gnet. Keras 3 only supports V3 `.keras` and `.weights.h5` files, or legacy V1/V2 `.h5` files.`
Do you have any hint?

**Reply by xiaohang007** [reply]
Dear Prof. Rignanese,
Thank you very much for your interest on SLICES. 
I haven't seen this error before, and solving this issue on Mac might be challenging. 
Fortunately, you can utilize our Docker image to run SLICES on a Mac. An English video tutorial is available at [here](https://www.bilibili.com/video/BV1Jx4y1S7ta/) covering how to set up the Docker environment, execute examples, and perform inverse design tasks. Additionally, a text-based tutorial can be found at [here](https://github.com/xiaohang007/SLICES?tab=readme-ov-file#tutorials)  for further guidance. 
You're welcome to reach out to me at any time.

**Reply by xiaohang007** [reply]
If you still would like to install SLICES on mac directly. Maybe you can try this:
conda create --name slices python=3.9
conda activate slices
pip install slices

**Reply by gmrigna** [reply]
Thanks for the reply. I have actually found a workaround in the meantime replacing M3GNet with MACE...
